astropy/table/pprint.py

```diff
@@ -213,15 +213,13 @@ def _get_pprint_size(max_lines=None, max_width=None):
             max_lines = lines
         elif max_lines < 0:
             max_lines = sys.maxsize
-        if max_lines < 8:
-            max_lines = 8
+        max_lines = max(max_lines, 8)
```

Comment:
On a second look after finding it was slower, I'm less sure about this case: it follows a whole slew of other size comparisons using `if`, so perhaps nicer to keep it as is?

Reply:
Hmm... I applied `benchmark` label FWIW.

Reply:
Please revert to the original for performance and readability. Note this continues the pattern of checks from above.